ReactWindows/ChakraBridge/JsModulesUnbundle.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <stdint.h>
+
+struct JsModulesUnbundleModule sealed
+{
+public:
+	wchar_t* source;
+	wchar_t* sourceUrl;
+	~JsModulesUnbundleModule()
+	{
+		delete[] source;
+		delete[] sourceUrl;
+	}
+};
+
+class JsModulesUnbundle
+{
+public:
+    JsModulesUnbundle(const wchar_t* sourcePath);
+	~JsModulesUnbundle();
+	virtual JsModulesUnbundleModule* GetModule(uint32_t index);
```

Comment:
would it be more useful if you had the JsModulesUnbundleModule as an out parameter and returned some error code?